addons/nuxeo-platform-pdf-utils/src/main/java/org/nuxeo/ecm/platform/pdf/operations/PDFExtractTextOperation.java

```diff
@@ -39,8 +41,9 @@
  *
  * @since 8.10
  */
-@Operation(id = PDFExtractTextOperation.ID, category = Constants.CAT_DOCUMENT, label = "PDF: Extract Text",
-    description = "Extracts raw text from a PDF. If the PDF is encrypted, a password is required.")
+@Operation(id = PDFExtractTextOperation.ID, category = Constants.CAT_DOCUMENT, label = "PDF: Extract Text", description = "Extracts raw text from a PDF."
+        + " If the PDF is encrypted, a password is required."
+        + " If patterntofind is not provided, extracts all the text it can.")
```

Comment:
While you're editing this, could you add something like `The extracted text is then set in the targetxpath property of the input document`? That's not obvious and quite important I would think. You could also mention the removepatternfromresult property maybe...

Reply:
ok, doing it

Reply:
(explaining all parameters)

Reply:
(done. I will squash once approved)